packages/office-ui-fabric-react/src/components/Button/BaseButton.classNames.ts

```diff
@@ -14,7 +14,7 @@ export interface IButtonClassNames {
   screenReaderText?: string;
 }
 
-const GlobalClassNames = {
+export const GlobalClassNames = {
```

Comment:
oh ummm can we export these as something specific to button?

Reply:
Made the change... as soon as I have the green light I will merge as I am constantly solving conflicts.

Reply:
Hmmm.... actually I need @mikewheaton or @yiminwu approval too now that I chaged those exports 😢